netlify.toml

```diff
@@ -4,23 +4,71 @@
   status = 301
 
 [[redirects]]
-  from = "/"
-  to = "/docs/welcome/overview/"
+  from = "/docs/enterprise/*"
+  to = "/docs/self-hosted/:splat"
   status = 301
 
+# Before version 1.4, we used to point the homepage to `/welcome/overview`.
+# These redirects make sure we don’t show a 404
+
 [[redirects]]
-  from = "/docs/enterprise/*"
-  to = "/docs/self-hosted/:splat"
+  from = "/docs/welcome/overview/"
+  to = "/docs/"
+  status = 301
+
+[[redirects]]
+  from = "/docs/1.4/welcome/overview/"
+  to = "/docs/"
+  status = 301
+
+[[redirects]]
+  from = "/docs/1.3/welcome/overview/"
+  to = "/docs/1.3"
+  status = 301
+
+[[redirects]]
+  from = "/docs/1.2/welcome/overview/"
+  to = "/docs/1.2"
+  status = 301
+
+[[redirects]]
+  from = "/docs/1.1/welcome/overview/"
+  to = "/docs/1.1"
+  status = 301
+
+[[redirects]]
+  from = "/docs/1.0/welcome/overview/"
+  to = "/docs/1.0"
+  status = 301
+
+[[redirects]]
+  from = "/docs/0.15/welcome/overview/"
+  to = "/docs/0.15"
+  status = 301
+
+[[redirects]]
+  from = "/docs/0.14/welcome/overview/"
+  to = "/docs/0.14"
+  status = 301
+
+[[redirects]]
+  from = "/docs/0.14/welcome/overview/"
+  to = "/docs/0.13"
+  status = 301
+
+[[redirects]]
+  from = "/docs/0.12/welcome/overview/"
+  to = "/docs/0.12"
   status = 301
 
 [[redirects]]
-  from = "/docs/:version/*"
-  to = "/docs/:version/welcome/overview/"
+  from = "/docs/0.11/welcome/overview/"
+  to = "/docs/0.11"
   status = 301
 
 [[redirects]]
-  from = "/docs/:version/*"
-  to = "/docs/:version/welcome/overview/"
+  from = "/docs/1.4/*"
```

Comment:
Is this something we'll need to change on every new doc version?

Reply:
Only this one because we point people at `okteto.com/docs/1.4/XXX` at the moment on the docs site. We’ll be able to remove the redirect when 1.5 points to `okteto.com/docs` and not have to worry about it after

Reply:
BUT thinking more about it:

Since we use `/docs/1.5/` for the next version, I guess when we point 1.5 to `/docs`, yeah we should update this to redirect `/docs/1.5` to `/docs`

I don’t know how much people bookmarks docs and all, specially in unreleased versions, but it wouldn’t hurt to update it.

Reply:
BUT thinking more about it:

Since we use `/docs/1.5/` for the next version, I guess when we point 1.5 to `/docs`, yeah we should update this to redirect `/docs/1.5` to `/docs`

I don’t know how much people bookmarks docs and all, specially in unreleased versions, but it wouldn’t hurt to update it.

Reply:
would this be part of the "release a new version" task list?

Reply:
Yep!